PokeOneWeb.DataSync/ReadModelUpdate/Impl/ReadModelUpdateService.cs

```diff
@@ -90,6 +96,10 @@ public void UpdateReadModel(SpreadsheetImportReport importReport)
             _itemRepository.Update(_itemMapper.MapFromDatabase(importReport));
             _reporter.StopReadModelUpdate("items");
 
+            _reporter.StartReadModelUpdate("regions");
+            _regionRepository.Update(_regionMapper.MapFromDatabase(importReport));
+            _reporter.StopReadModelUpdate("region");
```

Comment:
String used in StartReadModelUpdate() and StopReadModelUpdate() method calls does not match ("regions" vs. "region")